lrtable/src/lib/itemset.rs

```diff
@@ -199,8 +205,21 @@ mod test {
     }
 
     fn eco_grammar() -> YaccGrammar {
+        let mut header = Header::new();
+        header
+            .contents_mut()
+            .set_merge_behavior(&"yacckind".to_string(), MergeBehavior::Ours);
+        header.contents_mut().mark_required(&"yacckind".to_string());
+        // FIXME seems like this this should be Eco!
```

Comment:
Searching through this patch for `eco`, it appears that there are
a couple more instances where the yacc kind seems like it should be `Eco`
but at some point most likely was accidentally changed to `GenericParseTree`.

Reply:
It appears this one at least has been this way since the beginning of lrtable.
I'll look into the others I see.